src/main/kotlin/no/nav/familie/oppdrag/config/ØkonomiConsumerConfig.kt

```diff
@@ -0,0 +1,74 @@
+package no.nav.familie.oppdrag.config
+
+import no.nav.common.cxf.STSConfigurationUtil
+import no.nav.common.cxf.StsConfig
+import no.nav.okonomi.tilbakekrevingservice.TilbakekrevingPortType
+import org.apache.cxf.configuration.jsse.TLSClientParameters
+import org.apache.cxf.ext.logging.LoggingFeature
+import org.apache.cxf.ext.logging.LoggingInInterceptor
+import org.apache.cxf.ext.logging.LoggingOutInterceptor
+import org.apache.cxf.frontend.ClientProxy
+import org.apache.cxf.jaxws.JaxWsProxyFactoryBean
+import org.apache.cxf.transport.http.HTTPConduit
+import org.apache.cxf.ws.addressing.WSAddressingFeature
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+import javax.xml.namespace.QName
+
+@Configuration
+class ØkonomiConsumerConfig(
+        @Value("\${SECURITYTOKENSERVICE_URL}") private val stsUrl: String,
+        @Value("\${CREDENTIAL_USERNAME}") private val systemuserUsername: String,
+        @Value("\${CREDENTIAL_PASSWORD}") private val systemuserPwd: String,
+        @Value("\${TILBAKEKREVING_V1_URL}") private val tilbakekrevingUrl: String) {
+
+    private val WSDL = "wsdl/no/nav/tilbakekreving/tilbakekreving-v1-tjenestespesifikasjon.wsdl"
+    private val NAMESPACE = "http://okonomi.nav.no/tilbakekrevingService/"
+    private val SERVICE = QName(NAMESPACE, "TilbakekrevingService")
+    private val PORT = QName(NAMESPACE, "TilbakekrevingServicePort")
+
+    @Bean
+    fun økonomiService(): TilbakekrevingPortType {
```

Comment:
Det var mye saker for dette oppsettet, er det sånn att vi kan gjennbruke noe liknende som er i `familie-integrasjoner`? 
https://github.com/navikt/familie-integrasjoner/blob/master/src/main/java/no/nav/familie/integrasjoner/config/ServiceConfig.kt

```
CXFClient(PersonV3::class.java)
                    .address(personV3Url)
                    .timeout(20000, 20000)
                    .configureStsForSystemUser(stsConfig())
                    .build()
```

Eller går vi miste av det som eks er i `disableCnCheck` ?

Reply:
nei, tror jeg, Denne konfigen fungerte i familie-tilbake. Så jeg flyttet den bare fra tilbake til oppdrag

Reply:
Er sikkert fint å prøve å sette den opp på lik måte som i `ServiceConfig`, så trenger vi ikke unødvendig kode - hvis det virker uten dette med cnCheck som jeg har null peiling på

Reply:
Jeg husker jeg egentlig startet å prøve denne måten i begynnelsen i familie-tilbake. Men det fungerte ikke.